parsec/include/parsec/sys/atomic-c11.h

```diff
@@ -183,37 +183,42 @@ __int128_t parsec_atomic_fetch_add_int128(volatile __int128_t* l, __int128_t v)
 #endif
 
 /* Locks */
-
-typedef volatile atomic_flag parsec_atomic_lock_t;
+typedef union {
```

Comment:
This is ugly. What if you remove the volatile completely ?

Reply:
the volatile is not useful I removed it and it passes all tests